src/Output/PhpunitLike.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Asynit\Output;
+
+use Asynit\Test;
+use Symfony\Component\Console\Formatter\OutputFormatterStyle;
+
+class PhpunitLike implements OutputInterface
+{
+    const SPLIT_AT = 60;
+    const MAX_TRACE = 10;
+
+    private $outputFormatFail;
+    private $outputFormatSuccess;
+    private $outputFormatSkipped;
+    private $testOutputed;
+    private $failures;
+    private $assertionCount;
+
+    public function __construct()
+    {
+        $this->outputFormatFail = new OutputFormatterStyle('red', null, ['bold']);
+        $this->outputFormatSuccess = new OutputFormatterStyle('default', null, ['bold']);
+        $this->outputFormatSkipped = new OutputFormatterStyle('cyan', null, ['bold']);
+
+        fwrite(STDOUT, "Asynit Test suite\n\n");
```

Comment:
it just that i'm not a fan of passing a object from class to class where it can be a simple fwrite / echo, but can be changed latter don't really care about this